protobuf/bess_msg.proto

```diff
@@ -522,8 +519,8 @@ message ListGateHooksResponse {
 // Enable/Disable a global ResumeHook.
 message ConfigureResumeHookRequest {
   string hook_name = 1;         /// Name of the hook
-  bool enable = 2;              /// Installs the hook if True, else uninstalls
-  google.protobuf.Any arg = 3;  /// Hook-specific arguments
+  bool enable = 5;              /// Installs the hook if True, else uninstalls
+  google.protobuf.Any arg = 6;  /// Hook-specific arguments
 }
```

Comment:
Is there any reason for changing the field number in a way that not sequential? I do not exactly know how internal gRPC works, but it should probably fine as they are unique. Just very personal curious.

Reply:
Nice catch. I deleted a field. I will make it sequential.

Reply:
No worry!

Reply:
Actually its because I patched from the pangolin bess_msg.proto, which is wrong. That was caused by someone else. I fixed both anyway.

Reply:
There's a gap left over in some protobufs in bess from when those field numbers meant something else. The general idea is that if you delete something you might leave in a comment, something like "// 4: was foo, now reserved" and avoid re-using it so that anyone still using an old protobuf definition will send field 4 which you'll ignore, rather than using as something new.

We haven't been very good about this. 😺